src/applications/ivc-champva/10-10D/pages/ApplicantRelationshipPage.jsx

```diff
@@ -65,7 +65,10 @@ function generateOptions({ data, pagePerItemIndex }) {
     marriageOptions.push({ label: marriedDeceased, value: 'spouse' });
   } else {
     marriageOptions.push({ label: marriedLiving, value: 'spouse' });
-    marriageOptions.push({ label: marriedLivingDivorced, value: 'spouse' });
+    marriageOptions.push({
+      label: marriedLivingDivorced,
+      value: 'spouseSeparated',
+    });
```

Comment:
This is a bug fix. Previously, two objects each with value 'spouse' were pushed into the array, which caused rendering issues. This fixes that.